src/main/java/net/marcellperger/mathexpr/parser/Parser.java

```diff
@@ -81,11 +80,13 @@ public MathSymbol parseInfixPrecedenceLevel(int level) throws ExprParseException
         if(level == 0) {
             return parseParensOrLiteral();
         }
-
+        // TODO remove this next-line:
+        if(SymbolInfo.PREC_TO_INFO_MAP.get(level) == null) { return parseInfixPrecedenceLevel(level - 1); }
```

Comment:
```suggestion
```

Reply:
Although it might be useful to allow empty slots. In any case, we need to think about how we want to do this.